iOS/ExampleSuperApp.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
If Package.resolved is not included in the repo, Xcode should create it using the latest available versions of the dependencies acording to it's restrictions (which I would recommend using upToNextMajorVersion).